### PR TITLE
Feature/menu registration api

### PR DESCRIPTION
**Pull request by ppinkypeach**
1. delete update _api
2. improve menu_api

### CODE REVIEW

File: backend/models/crud.py [hunk 1]
```diff
@@ -44,39 +46,29 @@ def create_user_item(db: Session, item: schemas.ItemCreate, user_id: int):
 
 #################추가###################################################################
```
Comment (KKodiac) [inline backend/models/crud.py:47]: Please remove unnecessary comments😀

File: backend/models/schemas.py [hunk 2]
```diff
@@ -40,27 +41,28 @@ class Config:
 ###############추가
```
Comment (KKodiac) [inline backend/models/schemas.py:41]: 😀

**Review by KKodiac — CHANGES_REQUESTED** [review]
Request removing unnecessary comments

**Review by edi54 — APPROVED** [review]
수고하셨습니다!!

**Comment by juwon5272** [comment]
고생하셨어요

**Pull request merged**